DevExtreme.AspNet.TagHelpers.Generator/ChartSeriesSpecificSettings/Entry.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace DevExtreme.AspNet.TagHelpers.Generator.ChartSeriesSpecificSettings {
```

Comment:
It seems to be strange to use one approach for one XML-dependent case, and another approach for the other case.
Moreover, why extra `System.Xml.XDocument`, `System.Xml.XPath` references should be added?